blog/content/post/2021-10-10-optiplex-dasharo.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Dasharo for Dell OptiPlex 7010 / 9010'
+abstract: 'Open source firmware may be hundred times better than the
+           proprietary one. On the example of Dell OptiPlex 7010 / 9010 we will
+           show you the advantages of Dasharo firmware on this machine.'
+cover: /covers/dasharo-sygnet.svg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-10-10
+archives: "2021"
+
+tags:
+  - firmware
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+Dell OptiPlex 7010 / 9010 is the majority of machines used in 3mdeb office by the
+engineers and developers. It is only natural for a company propagating open
+source firmware (OSFW) to utilize their own product in daily work. The process
+of porting coreboot on the machine took significant amount of free time (which
+you may read about in [other blog posts](https://blog.3mdeb.com/2020/2020-06-24-dell-optiplex-port/)
```

Comment:
@miczyg1 maybe we can point to tag that mark both optiplex posts?

Reply:
Added new tags to the Optiplex posts and now the URL should link to them all

Reply:
Ok, in long run maybe consistency between Dasharo, coreboot and our blog post, naming could be achieved.